app/models/feed.rb

```diff
@@ -54,6 +54,7 @@ class Feed < ApplicationRecord
   validates :enclosure_prefix, http_url: true
   validates :display_episodes_count, numericality: {only_integer: true, greater_than: 0}, allow_nil: true
   validates :display_full_episodes_count, numericality: {only_integer: true, greater_than: 0}, allow_nil: true
+  validates :description, bytesize: {maximum: Episode::MAX_DESCRIPTION_BYTES}
```

Comment:
As I said in slack, I checked production data, and there are no feed/podcast descriptions over the 4k limit, so is safe to impose retroactively.